src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -672,6 +672,15 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
                     return;
                 }
 
+                foreach ((string, DataType, object) element in context.Inputs)
+                {
+                    if (element.Item3.GetType() == typeof(ParameterBindingData) || element.Item3.GetType() == typeof(ParameterBindingData[]))
```

Comment:
Question: Do we care about collecting this information for every function invocation at runtime? If we are interested in usage of sdk binding usage in function apps, could we not infer that from the function metadata during worker startup/init?

Reply:
Yeah I think we might be able to log when parsing the metadata to determine the supports deferred binding flag and use that as an indication that this function should be using ParameterBindingData. Keeping in mind that not every function in the Function App will be using `ParameterBindingData`, just the ones who use the new blob ext so we will need to capture for each function.

I think that should be enough on the telemetry front. But for logging/debug purposes, we can log different parts of the process to see whats happening with the data as we handle it

Reply:
I agree with Shyju that we shouldn't be logging this on each invocation, since that has performance implications. It should done at index/load type time. Also we shouldn't need both a log statement and a metric event right? Can't the metric event suffice?

Reply:
Moved telemetry to Load request instead of per invocation.